xml/wgl.xml

```diff
@@ -766,6 +766,7 @@ Registry at https://github.com/KhronosGroup/OpenGL-Registry
         <command>
             <proto><ptype>UINT</ptype> <name>wglEnumerateVideoCaptureDevicesNV</name></proto>
             <param><ptype>HDC</ptype> <name>hDc</name></param>
+            <!-- FIXME: Some way for len="" to refer to the return value. Maybe len="RETURN()"? -->
```

Comment:
@SunSerega what do you think about this?
Should we introduce `RETURN()` or should we just leave this unhandled?

Reply:
I wouldn't mind either way, but also while this is the only case - it doesn't matter much, since this case would need to be handled manually anyway, and IMO `len` attribute values are not yet consistent enough to use as notification for consumers, like "some new, yet unexpected value appeared, need to figure out how to handle this". Mainly because of all the `COMPSIZE`-s.

Reply:
Fair enough, I could remove this comment or leave it in. 

I'll leave it to @oddhack to say whether this comments gets to stay or gets removed.